client/src/types/error.rs

```diff
@@ -42,6 +42,9 @@ pub enum ClientError {
 
     #[error("Connection failure ({0})")]
     ConnectionFailure(String),
+
+    #[error("Snapshot file is missing ({0})")]
+    SnapshotfileMissing(String),
```

Comment:
In case it was not on purpose
```suggestion
    SnapshotFileMissing(String),
```